articles/third-party/third-ref-eula.md

```diff
@@ -23,10 +23,11 @@ Product | EULA
 --------|-----
 Cisco AppDynamics | <https://www.appdynamics.com/download-eula/>
 CommVault Complete Backup & Recovery | <https://documentation.commvault.com/commvault/v11_sp9/article?p=features/misc/eula.htm>
+CyberScore | N/A
+Hystax Acura Cloud Migration | [EULA](images/third-hystax-eula.pdf)
```

Comment:
Can we point to here instead of using a PDF file? https://hystax.com/terms-of-use/

(if the terms change, the PDF will become out of date)

Reply:
Hi Sue, great find! Yes, this is better to use than the PDF originally supplied